analyzers/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -50,10 +50,10 @@
 
   <Target Name="CopyBinaries" AfterTargets="Build" DependsOnTargets="SignDlls" Condition=" '$(TargetFramework)' == 'NET46' ">
     <ItemGroup>
+      <BinariesToCopy Include="$(OutputPath)\Google.Protobuf.dll" />
```

Comment:
Why this change? The order in which the binaries are copied should not be important, right? If you just wanted to make sure to have a consistent order for both languages than maybe it is also worth it having the same order of steps for the install and uninstall scripts for both languages...

Reply:
Alphabetical order for readability. Random sort was hard to read/validate.